src/pages/NewPlaylist/index.jsx

```diff
@@ -1,11 +1,56 @@
-import React from "react";
+import React, { useState } from "react";
+import "./styles.scss";
+import { makeStyles } from "@material-ui/core/styles";
+import TextField from "@material-ui/core/TextField";
+import Button from "@material-ui/core/Button";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    "& > *": {
+      margin: theme.spacing(1),
+      width: "50ch",
+      borderColor: "white",
+    },
+  },
+}));
+
+const NewPlaylist = () => {
+  const classes = useStyles();
+  const [input, setInput] = useState();
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    console.log(input);
```

Comment:
I suggest we control whether input is empty/without wording chars (ex: if someone names his playlist (-'&""'(-_-('"), do we let him pass ?)

Reply:
Also could redirect to /temp-playlist just for the sake of test ?

Reply:
I suggest special characters allowed but the request is only triggered if the input is not an empty string. For the test link, I can add that